services/twitter.py

```diff
@@ -21,13 +21,20 @@ def plugin(srv, item):
     )
 
```

Comment:
it is unfortunate, that you are mixing two different things in one pull request; please try to avoid that in future. (owntracks in samplefuncs.py has nothing to do with twitter.py)

Reply:
Yes, i understand that.